This patch fix compatibility between scummvm/forbidden security enforcement
and fortify-headers. The first is meant to avoid direct access to system
calls for scummvm to be fully portable on a lot of platforms.

--- a/common/scummsys.h
+++ b/common/scummsys.h
@@ -464,6 +464,4 @@

 #endif

-#include "common/forbidden.h"
-
 #endif

